server/bundles/io.cloudbeaver.service.security/src/io/cloudbeaver/auth/provider/rp/RPAuthProvider.java

```diff
@@ -92,6 +94,11 @@ public DBWUserIdentity getUserIdentity(
             userMeta.put(SMStandardMeta.META_LAST_NAME, lastName);
         }
 
+        if (CommonUtils.isNotEmpty(fullName)) {
+            nameBuilder = new StringBuilder(fullName);
```

Comment:
this doesn't make sense since we don't support this meta information on UI side, we should use only first/last name